requirements.txt

```diff
@@ -1,119 +1,3 @@
-# Required for Home Assistant core
-requests>=2.0
```

Comment:
Also add `pip3 install -r requirements_all.txt` to our `.travis.yml` file to solve the test issues.